docs/tutorial/VueJS/step6.md

```diff
@@ -154,7 +162,6 @@ data: {
 	...
```

Comment:
Changing this also to 

data : function() {
        return {
                currentenv: null,
                SugarL10n: null,
                SugarPresence: null,
                ...

Reply:
done, thanks